raiden_contracts/tests/test_channel_close.py

```diff
@@ -439,6 +454,7 @@ def test_close_channel_event_no_offchain_transfers(
         create_channel,
         event_handler,
 ):
+    """ closeChannel succeeds and emits an event even with no balance proofs """
```

Comment:
I would suggest:
`closeChannel succeeds when called no balance proof and emits an event even with nonce 0`